recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "5.1.1" %}
+{% set version = "4.5.3" %}
```

Comment:
I'm OK merging this on `master` if it is a one-time thing, if you are not going to maintain this old version, and let the bot "fix it" later.